application/comit_node/src/bin/comit_node.rs

```diff
@@ -18,15 +15,15 @@ use comit_node::{
     },
 };
 use directories;
-use ethereum_support::*;
```

Comment:
how was this not caught by rust warnings?

Reply:
It imported the re-exported `Future` trait through that, that is why it was not obsolete.